package.json

```diff
@@ -5,13 +5,10 @@
   "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "./script/build",
```

Comment:
Why get rid of this one? I like being able to do a build in a way that's consistent across projects, particularly when using `npm link` (or I guess `yarn link` now).